quadriga/src/main/webapp/WEB-INF/views/layouts/main.html

```diff
@@ -50,6 +50,21 @@
 	          		<a th:href="@{/admin/user/list}" >Users</a>
 	          	</li>
 	          	
+	          	<li sec:authorize="hasRole('USER')" role="presentation">
```

Comment:
the drop down menu button shouldn't look like it's selected. It should have a white background like the others and only get a grey background on hover.